.expeditor/verify.scaffolding-chef-infra.yml

```diff
@@ -113,6 +122,15 @@ steps:
           privileged: true
     timeout_in_minutes: 60
 
+  - label: "[scaffolding-chef-inspec] :linux: :habicat: Run and test InSpec profile with API changes (multi-profile)"
```

Comment:
You don't need to run each of the test scenarios for this change you only need one test scenario for Linux and one for Windows that uses the multi-profile pattern. It is also totally fine if it's using the default profile set that only has one test and the rest of the variables are default.

Reply:
All set -- pulled out the tests for multi-profile other than the defaults for linux/windows